app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -25,6 +25,8 @@ def call
       def create_authentication_parameters
         @logger.debug(LogMessages::Authentication::AuthnJwt::CREATING_AUTHENTICATION_PARAMETERS_OBJECT.new)
         @authentication_parameters = Authentication::AuthnJwt::AuthenticationParameters.new(@authenticator_input)
+        @logger.debug(LogMessages::Authentication::AuthnJwt::FetchingTokenString.new)
+        @authentication_parameters.jwt_token = @jwt_configuration.create_authentication_parameters(@authentication_parameters)
```

Comment:
i would move the `jwt_token` to the constructor instead of using `attr_accessor`.

Reply:
oh damn i now see that the `authentication_parameters` contains a `jwt_token` to contains the `authentication_parameters`. is that correct?

Reply:
The change is related to previous PR.